src/native_driver.jl

```diff
@@ -47,28 +50,28 @@ available.
 """
 function K8sClusterManager(np::Integer;
                            namespace::String=current_namespace(),
-                           manager_pod_name::String=get(ENV, "HOSTNAME", "localhost"),
-                           image=nothing,
+                           manager_hostname::String=get(hostname, ENV, "HOSTNAME"),
+                           image="julia:$VERSION",
```

Comment:
This currently is a breaking change. We could possibly do something to have different defaults depending on whether this code is being run inside or outside of a cluster.